ensime-model.el

```diff
@@ -180,6 +180,14 @@
 (defun ensime-type-fqn (type)
   (plist-get type :fqn))
 
+(defun ensime-type-source-position (type)
```

Comment:
I'm generally not very keen on these things... I'd rather we used common lisp style destructuring where possible. They are just repetitive.